godot-macros/src/docs.rs

```diff
@@ -39,52 +39,62 @@ pub fn make_inherent_impl_docs(
     constants: &[ConstDefinition],
     signals: &[SignalDefinition],
 ) -> TokenStream {
+    /// Generates TokenStream containing field definitions for documented methods and documentation blocks for constants and signals.
     fn pieces(
         functions: &[FuncDefinition],
         signals: &[SignalDefinition],
         constants: &[ConstDefinition],
-    ) -> Option<TokenStream> {
+    ) -> TokenStream {
+        let to_tagged = |s: String, tag: &str| match s.is_empty() {
+            true => s,
+            _ => format!("<{tag}>{s}</{tag}>"),
+        };
```

Comment:
`if`

Reply:
fixed

Reply:
Not fixed in the diff. Did you forget to push?